src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceOndeDay = 40;
+
+  if (days >= 7) {
```

Comment:
move `7`, `3`, `50`, `20`  to variables to avoid using `magic` numbers 
https://www.inf.unibz.it/~calvanese/teaching/04-05-ip/lecture-notes/uni04/node15.html

You can move `days * priceOndeDay` to a variable as well